gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -1,20 +1,97 @@
 <script lang="ts">
-  import { parseSpans } from './parseSpans';
+  import { parseSpans, Span } from './parseSpans';
```

Comment:
Doesn't work for me unless I import `Span` as a type.

Reply:
fixed on main